generators/dotnet-templates/Microsoft.BotFramework.CSharp.CoreBot/content/CoreBot/Program.cs

```diff
@@ -13,16 +13,19 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            CreateWebHostBuilder(args).Build().Run();
+            CreateHostBuilder(args).Build().Run();
         }
 
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .ConfigureLogging((logging) =>
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
```

Comment:
If these are 3.1 changes we're going to need to make changes to account for 2.2.  so this may make this more work than is currently completed.

Reply:
Agreed - if we want to provide the choice for both then we need to add conditionals throughout, but I am unsure of the benefits vs the cost here.